tests/test_helpers/test_dialogue/test_base.py

```diff
@@ -280,102 +257,331 @@ def test_dialogue_properties(self):
-        invalid_message_id = DefaultMessage(
+    def test_is_message_by_self(self):
+        """Test the 'is_message_by_self' method."""
+        message_1 = DefaultMessage(
             dialogue_reference=(str(1), ""),
-            message_id=0,
+            message_id=1,
             target=0,
             performative=DefaultMessage.Performative.BYTES,
             content=b"Hello",
         )
-        invalid_message_id.counterparty = self.opponent_address
+        message_1.sender = self.agent_address
+        message_1.to = self.opponent_address
 
-        assert not self.dialogue.update(invalid_message_id)
-        assert self.dialogue.last_outgoing_message is None
+        message_2 = DefaultMessage(
+            dialogue_reference=(str(1), ""),
+            message_id=1,
+            target=0,
+            performative=DefaultMessage.Performative.BYTES,
+            content=b"Hello",
+        )
+        message_2.sender = self.opponent_address
+        message_2.to = self.agent_address
 
-    def test_update_self_initiated_dialogue_label_on_message_with_complete_reference_negative_not_second_message(
-        self,
-    ):
-        """Negative test for the '_update_self_initiated_dialogue_label_on_message_with_complete_reference' method: input message is not the second message in the dialogue."""
+        assert self.dialogue.is_message_by_self(message_1)
+        assert not self.dialogue.is_message_by_self(message_2)
+
+    def test_is_message_by_other(self):
+        """Test the 'is_message_by_other' method."""
+        message_1 = DefaultMessage(
+            dialogue_reference=(str(1), ""),
+            message_id=1,
+            target=0,
+            performative=DefaultMessage.Performative.BYTES,
+            content=b"Hello",
+        )
+        message_1.sender = self.agent_address
+        message_1.to = self.opponent_address
+
+        message_2 = DefaultMessage(
+            dialogue_reference=(str(1), ""),
+            message_id=1,
+            target=0,
+            performative=DefaultMessage.Performative.BYTES,
+            content=b"Hello",
+        )
+        message_2.sender = self.opponent_address
+        message_2.to = self.agent_address
+
+        assert not self.dialogue.is_message_by_other(message_1)
+        assert self.dialogue.is_message_by_other(message_2)
+
+    def test_try_get_message(self):
+        """Test the 'try_get_message' method."""
+        assert self.dialogue.try_get_message(1) is None
+
+        message_1 = DefaultMessage(
+            dialogue_reference=(str(1), ""),
+            message_id=1,
+            target=0,
+            performative=DefaultMessage.Performative.BYTES,
+            content=b"Hello",
+        )
+        message_1.sender = self.agent_address
+        message_1.to = self.opponent_address
+
+        self.dialogue.update(message_1)
+
+        assert self.dialogue.try_get_message(2) is None
```

Comment:
Magic values like the 2 here make tests hard to read for others. Declare a var like `unused_message_id = 2` and the next person will get it.